### ISSUE TITLE
dynamic-import-chunkname does not handle webpackInclude/webpackExclude comments correctly

### ISSUE THREAD

**Issue by vkrol** [import-js/eslint-plugin-import#1407]
`dynamic-import-chunkname` rule does not handle `webpackInclude`/`webpackExclude` comments correctly.

`.eslintrc.js`:
```js
module.exports = {
	parser: "babel-eslint",
	parserOptions: {
		ecmaVersion: 2018,
		sourceType: "module",
	},
	plugins: ["import"],
	rules: {
		"import/dynamic-import-chunkname": "error",
	},
};
```
`index.js`
```js
import(
	/* webpackChunkName: "foo" */
	/* webpackInclude: /foo/ */
	"./foo"
);
```
Output:
```
  1:1  error  dynamic imports require a leading comment in the form /* webpackChunkName: "[0-9a-zA-Z-_/.]+",? */  import/dynamic-import-chunkname

✖ 1 problem (1 error, 0 warnings)
```

Package name|Package version
-|-
`eslint` | `6.0.1`
`eslint-plugin-import` | `2.18.0`

Minimal reproducible example: https://github.com/vkrol/eslint-plugin-import-webpack-comment-with-regex-bug

**Comment by ljharb** [comment]
I suspect we didn’t know that commenting format existed when the rule was written :-)

**Comment by lencioni** [comment]
This rule was originally written when the only magic comment was `webpackChunkName`. Sounds like it needs to be updated to include newer webpack features.

**Comment by heinzdmx** [comment]
I do not know if this is inside the scope of this issue, but it appears that this rules also does not allow `[request]` or `[index]` which is also supported in webpackChunkName
Currently have to disable the rule the places where my code base uses them.

**Comment by vkrol** [comment]
It was fixed in https://github.com/import-js/eslint-plugin-import/pull/2330/.
